data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,49 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    description_short: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    description_long: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    load_fields:
+      - short_name: "ex_loss"
+        df_field_name: "RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME"
+        long_name: "FEMA Risk Index Expected Annual Loss Score"
+        field_type: float
+        include_in_tiles: true
+        include_in_csv: true
+        include_in_excel: true
+        column_position: 1
```

Comment:
Just two questions here: 
1. When would we have something in the csv but not in the excel? Do we need both controls?
2. And, is the code in the ETL going to be reworked entirely to get rid of `field_names.py` and `constants.py`? I'm just a bit nervous because it seems like we are going to have to carry this refactor all the way through the etl code and it's already August 5.

Reply:
@mattbowen-usds -- good question. I think the answer is that the column order applies *across* data sources that come from different ETL classes. So in this NRI class, we might set columns 7, 11, 18, 18.5, and 19, and in another class we set columns 8, 9.5, 12, etc. 

@emma-nechamkin good point in that `include_in_csv` and `include_in_excel` could probably be collapsed into a single boolean: `include_in_csv_and_excel`. 

I'm not sure the answer to @emma-nechamkin 's Question #2 -- that's a good one. @esfoobar-usds ?

Reply:
About point 2., we decided to wait until the new indicators are done, and then do the score and score post generation refactor using YAML.

About the column order, in our existing CSV and Excel YAML files that drive the column order for the side effects, it is assumed the column order is the list order. However, datasets YAML could have fields that are not going to be included in the downloadable assets, so we'd need to signify that. However, that flag can come from the `include_in_csv_and_excel` column, which I would rename as `include_in_downloadable_files`, so once again, I think we might be able to just make the list order be the column order. Let's discuss in the meeting today.

Reply:
After a conversation with the content stakeholder, @KameronKerger, we have settled on implicit column order within datasets and dataset order based on their position in the `datasets.yml` file. The index columns, like GEOID, county, DAC and demographic data will come first.